src/runtime-catalogue/RuntimeCatalogue.js

```diff
@@ -338,20 +356,15 @@ class RuntimeCatalogue {
 
         // create the descriptor
         let idpproxy = this._factory.createProtoStubDescriptorObject(
```

Comment:
Based on the specs here https://github.com/reTHINK-project/specs/tree/master/datamodel/core/hyperty-catalogue#protocol-stub-descriptor, the following parameters should be passed;
```js
rawProxy["constraints"],
rawProxy["hypertyType"],
rawProxy["dataObjects"],
rawProxy["interworking"],
rawProxy["idpProxy"],